### PR TITLE
List of recently checked in Qrtickets

### PR DESCRIPTION
**Pull request by Northo**
| image |
| :---: |
| Screenshot where you and me have been checked in (mobile emulation as I assume the scanner will mostly be used on mobile) |

### CODE REVIEW

**Review by kaprests — COMMENTED** [review]
Cool, seems like a really usefull feature/improvement!

The PR is still a draft, but should it be marked ready for review? If that's the case I'll approve :))

File: nablapps/contact/views.py [hunk 1]
```diff
@@ -6,8 +6,6 @@
 from django.shortcuts import render
 from django.urls import resolve
 
-from click import style
-
 from .forms import ContactForm, FeedbackForm, RoomForm
```
[the remark this answers is not in the document]
Comment (Northo) [reply nablapps/contact/views.py:9]: Looked like it...

**Comment by Northo** [comment]
@sondreklyve or @Amund211 , I added this in case the Qr ticket system will be used for jub. I think it makes the app slightly more usable.

(Sorry for the unrelated deletion line, will remove upon request).

File: nablapps/contact/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaprests) [reply nablapps/contact/views.py:9]: Yeh, strange that it hasn't been caught earlier, so nice find I guess! :^)

**Pull request merged**